### PR TITLE
Promote swipe gov for Polkadot And Kusama

### PR DESCRIPTION
**Pull request by ERussel**

### CODE REVIEW

File: chains/v20/chains.json [hunk 1]
```diff
@@ -107,6 +107,12 @@
                     "type": "subquery",
                     "url": "https://subquery-governance-polkadot-prod.novasama-tech.org"
                 }
+            ],
+            "referendum-summary": [
+                {
+                    "type": "novasama",
+                    "url": "https://opengov-backend-stg.novasama-tech.org/not-secure/api/v1/referendum-summaries/list"
```
Comment (leohar) [inline chains/v20/chains.json:114]: need to remove `not-secure/`

Comment (stepanLav) [reply chains/v20/chains.json:114]: I haven't changed it on stg yet, we can keep it as it is and move to prod version, when it will be deployed

Comment (stepanLav) [inline chains/v20/chains.json:114]:
```suggestion
                    "url": "https://opengov-backend.novasama-tech.org/api/v1/referendum-summaries/list"
```

Screenshot 2024-10-07 at 15 41 02

File: chains/v20/chains.json [hunk 2]
```diff
@@ -233,6 +239,12 @@
                     "type": "subquery",
                     "url": "https://subquery-governance-kusama-prod.novasama-tech.org"
                 }
+            ],
+            "referendum-summary": [
+                {
+                    "type": "novasama",
+                    "url": "https://opengov-backend-stg.novasama-tech.org/not-secure/api/v1/referendum-summaries/list"
```
Comment (stepanLav) [inline chains/v20/chains.json:246]:
```suggestion
                    "url": "https://opengov-backend.novasama-tech.org/api/v1/referendum-summaries/list"
```

**Review by leohar — APPROVED** [review]

**Review by stepanLav — APPROVED** [review]

**Pull request merged**